README.md

```diff
@@ -252,6 +252,32 @@ make container
 Simply build and run kube-state-metrics inside a Kubernetes pod which has a
 service account token that has read-only access to the Kubernetes cluster.
 
+### Prometheus Operator
+
+The ([`kube-prometheus`](https://github.com/prometheus-operator/kube-prometheus/)) stack installs kube-state-metrics as one of its [components](https://github.com/prometheus-operator/kube-prometheus#kube-prometheus); you do not need to install kube-state-metrics if you're using the kube-prometheus stack.
+
+The kube-state-metrics Deployment [manifest] overrides several command line arguments of the kube-state-metrics binary:
```

Comment:
```suggestion
The `kube-state-metrics` Deployment [manifest] overrides several command line arguments of the kube-state-metrics binary:
```

Reply:
This text has been remove (https://github.com/kubernetes/kube-state-metrics/pull/1412/commits/86689d45db2730aff799e495729f29993734087c)